package.json

```diff
@@ -3,43 +3,43 @@
+    "lint:fix": "turbo run lint -- --fix --continue",
+    "lint:style": "turbo run lint:style --continue",
+    "prepublishOnly": "turbo run prepublishOnly --continue",
+    "release": "turbo publish --continue",
+    "storybook:build": "turbo run storybook:build --continue",
+    "storybook:build-changed": "cross-env-shell CHANGED_ONLY=true turbo run storybook:build --include-dependencies --continue",
+    "storybook:serve": "turbo run storybook:serve --continue",
+    "storybook:serve-changed": "cross-env-shell CHANGED_ONLY=true turbo run storybook:serve --continue",
+    "storybook:serve-static": "turbo run storybook:serve-static --continue",
+    "storybook:deploy": "turbo run storybook:deploy --continue",
+    "test": "cross-env-shell \"turbo run $LERNA_ARGS test\" --continue",
+    "ci:test:tools": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:tools --continue\"",
+    "ci:test:services": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:services --continue\"",
+    "ci:test:atoms": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:atoms --continue\"",
+    "ci:test:molecules": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:molecules --continue\"",
+    "ci:test:organisms": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:organisms --continue\"",
+    "ci:test:pages": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:pages --continue\"",
+    "ci:test:templates": "cross-env-shell \"turbo run $LERNA_ARGS ci:test:templates --continue\"",
+    "ci:test-component:chrome:atoms": "cross-env-shell TEST_TYPE=component turbo run $LERNA_ARGS ci:test-component:chrome:atoms --concurrency=1 --continue",
+    "ci:test:visual:atoms": "cross-env-shell TEST_TYPE=visual turbo run $LERNA_ARGS ci:test:visual:atoms --concurrency=1 --continue",
+    "ci:test-a11y:chrome:atoms": "cross-env-shell TEST_TYPE=a11y turbo run $LERNA_ARGS ci:test-a11y:chrome:atoms --concurrency=1 --continue",
+    "ci:test-component:chrome:molecules": "cross-env-shell TEST_TYPE=component turbo run $LERNA_ARGS ci:test-component:chrome:molecules --concurrency=1 --continue",
+    "ci:test:visual:molecules": "cross-env-shell TEST_TYPE=visual turbo run $LERNA_ARGS ci:test:visual:molecules --concurrency=1 --continue",
+    "ci:test-a11y:chrome:molecules": "cross-env-shell TEST_TYPE=a11y turbo run $LERNA_ARGS ci:test-a11y:chrome:molecules --concurrency=1 --continue",
+    "ci:test-component:chrome:organisms": "cross-env-shell TEST_TYPE=component turbo run $LERNA_ARGS ci:test-component:chrome:organisms --concurrency=1 --continue",
+    "ci:test:visual:organisms": "cross-env-shell TEST_TYPE=visual turbo run $LERNA_ARGS ci:test:visual:organisms --concurrency=1--continue",
+    "ci:test-a11y:chrome:organisms": "cross-env-shell TEST_TYPE=a11y turbo run $LERNA_ARGS ci:test-a11y:chrome:organisms --concurrency=1 --continue",
+    "ci:test-component:chrome:pages": "cross-env-shell TEST_TYPE=component turbo run $LERNA_ARGS ci:test-component:chrome:pages --concurrency=1 --continue",
```

Comment:
The new `--continue` arguments ensures execution continues even if a package fails tests, resulting in exit code 1